apps/src/p5lab/gamelab/GameLab.js

```diff
@@ -1,6 +1,7 @@
 import P5Lab from '../P5Lab';
 import project from '@cdo/apps/code-studio/initApp/project';
 import {showLevelBuilderSaveButton} from '../../code-studio/header';
+import {workspace_running_background, white} from '@cdo/apps/util/color';
```

Comment:
nit: the convention is to import the whole color module `import color from '@cdo/apps/util/color';` then in the code use `color.white` and `color.workspace_running_background` which makes it a little more clear what the variables are/where they come from.